learn/guides/ballerina-shell.md

```diff
@@ -15,9 +15,12 @@ It allows you to evaluate snippets of code without having to write complete prog
 
 This will particularly be useful if you are looking to try out the language and its new features, and also for debugging purposes.
 
+>**Please Note:** Ballerina Shell currently supports subset of the language features. Ballerina Shell doesn't support 
+> services and isolation features. Also, Ballerina Shell provides limited feature support for the worker related implementations.
```

Comment:
```suggestion
> services and isolation features. Also, Ballerina Shell provides limited feature support for the worker-related implementations.
```

Reply:
Updated in 2a10777065a5455f4155b3fbdd75b7a70f169c75